File: 1-Patron/php/Experiencia.php
<?php
class Experiencia
{
  private $total;
  private $nueva;

  function __construct()
  {
    $this->total = 0;
    $this->nueva = 0;
  }

  function getTotal()
  {
    return $this->total;
  }

  function setTotal($total)
  {
    $this->total = $total;
  }

  function getNueva()
  {
    return $this->nueva;
  }

  function setNueva($nueva)
  {
    $this->nueva = $nueva;
  }

  function incTotal($incremento)
  {
    $this->total += $incremento;
  }

  function decNueva($decremento)
  {
    $this->nueva -= $decremento;
  }

  function incNueva($incremento)
  {
    $this->nueva -= $incremento;
  }
}
?>

File: 1-Patron/Java/Experiencia.java
package masterwholeproxy;

public class Experiencia {
    private int total;
    private int nueva; 
    public Experiencia(){
        this.total = 0;
        this.nueva = 0;
    }
    public int getTotal(){
        return this.total;
    }
    public int getNueva(){
        return this.nueva;
    }
    public void setTotal(int total){
        this.total = total;
    }
    public void setNueva(int nueva){
        this.nueva = nueva;
    }
    public void incTotal(int incremento){
        this.total += incremento; 
    }
    public void decNueva(int decremento){
        this.nueva -= decremento;
    }
    public void incNueva(int incremento){
        this.nueva = incremento;
    }
}
